### ISSUE TITLE
First time startup error

### ISSUE THREAD

**Issue by franc-ist** [Da-Vinci/Discord-MacDJ#2]
![](http://i.fishyfing.xyz/11luh.png)

**Comment by devsnek** [comment]
if you open and close it again this goes away. it has to do with a first time unhandled database creation error @FishyFing

**Comment by briantanner** [comment]
@FishyFing yeah, this is a known issue, but a workaround is if you close and restart it should work.
